locales/de/client.json

```diff
@@ -2068,7 +2068,6 @@
   "The User Manager will automatically create, update, and delete users as you invite players into your Game, a player leaves, or settings are modified.": "The User Manager will automatically create, update, and delete users as you invite players into your Game, a player leaves, or settings are modified.",
   "The world was restored to Save Point {{- title}}": "Die Welt wurde am Save Point wiederhergestellt {{- title}}",
   "The world was restored to the automatic backup from {{date}}": "Die Welt wurde mit der automatischen Sicherung von {{date}}",
-  "Theme selection is currently disabled. Click here for more information": "Theme selection is currently disabled. Click here for more information",
```

Comment:
This is removed as KaKaRoTo reverted the code associated with April 1st theme update